stable/admin/templates/_sidecar.tpl

```diff
@@ -89,4 +99,4 @@ shareProcessNamespace: true
   emptyDir: {}
```

Comment:
I thought that this actually needed to create a new volume, and I did not want to add more plumbing to create more generic ephemeral volumes. But I think this can actually use the existing volume with subPath.

Reply:
The volume is used for inter-container communication.